web/react/components/navbar_dropdown.jsx

```diff
@@ -223,6 +233,18 @@ export default class NavbarDropdown extends React.Component {
                                 {'Report a Problem'}
                             </a>
                         </li>
+                        <li>
+                            <a
+                                href='#'
+                                onClick={this.handleAboutModal}
+                            >
+                                {'About Mattermost'}
+                            </a>
+                        </li>
+                        <AboutBuildModal
+                            show={this.state.showAboutModal}
+                            onModalDismissed={this.aboutModalDismissed}
+                        />
```

Comment:
Is here an appropriate place for `AboutBuildModal`?

I'm not 100% familiar with React yet, but I have a feeling this might not be right. Only other alternative I can think of is putting it outside of the `ul`, but that would require wrapping in a `span`. Which option is more appropriate?